src/communication/index.ts

```diff
@@ -0,0 +1,29 @@
+import { SDKMessageToPayload, SDKMessageIds, SentSDKMessage, RequestId } from '../types';
+
+const sendMessageToInterface = <T extends SDKMessageIds, D = SDKMessageToPayload[T]>(
+  messageId: T,
+  data: D,
+  requestId?: RequestId,
+): SentSDKMessage<T, D> => {
+  if (!requestId) {
+    if (typeof window !== 'undefined') {
+      requestId = Math.trunc(window?.performance.now());
+    } else {
+      requestId = Math.trunc(Date.now());
+    }
+  }
+  const message = {
+    messageId,
+    requestId,
+    data,
+  };
+
+  if (typeof window !== 'undefined') {
+    window.parent.postMessage(message, '*');
+  }
+
+  return message;
+};
+
+export { sendMessageToInterface };
```

Comment:
what is the difference of this and putting `export` in front of `const sendMessageToInterface `

Reply:
from a technical perspective, there are none, it will be handled the same way. This way you could also rename exports like `export { sendMessageToInterface as sendMessage }`. Also this way you can get a better perspective of what is exported from a module without scanning every variable. For this case, it's all about personal preference

Reply:
ahhh ok, thanks for the explanation :)